web-wallet/src/routes/(app)/dashboard/send/__tests__/page.spec.js

```diff
@@ -1,6 +1,10 @@
-import { afterEach, describe, expect, it, vi } from "vitest";
-import { cleanup, render } from "@testing-library/svelte";
-import Send from "../+page.svelte";
+import { afterAll, afterEach, describe, expect, it, vi } from "vitest";
+import { cleanup, fireEvent, render } from "@testing-library/svelte";
+
+import mockedWalletStore from "../../../../../__mocks__/mockedWalletStore";
```

Comment:
I think at this point it's worth creating an alias for the mocks folder?

Reply:
https://github.com/dusk-network/rusk/issues/2849